static/css/subtitle.css

```
@@ -177,6 +327,79 @@ body.subtitle-window-host.subtitle-linux-host #subtitle-display {
+    inset: 0;
+    z-index: 2;
+    overflow: hidden;
+    pointer-events: none;
+    --subtitle-danmaku-edge-fade: 18px;
+    -webkit-mask: linear-gradient(
+        90deg,
+        transparent 0,
+        #000 var(--subtitle-danmaku-edge-fade),
+        #000 calc(100% - var(--subtitle-danmaku-edge-fade)),
+        transparent 100%
+    );
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)